src/coreclr/src/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Windows.props

```
@@ -78,6 +74,7 @@ The .NET Foundation licenses this file to you under the MIT license.
       <LinkerArg Condition="'$(OutputType)' == 'Exe'" Include="/SUBSYSTEM:CONSOLE" />
       <LinkerArg Condition="'$(OutputType)' == 'WinExe' or '$(OutputType)' == 'Exe'" Include="/ENTRY:$(IlcProcessEntrypoint)" />
       <LinkerArg Include="/NATVIS:&quot;$(MSBuildThisFileDirectory)CoreRTNatVis.natvis&quot;" />
+      <LinkerArg Include="/wholearchive:$(BootstrapperLibraryName).lib" />
```

```suggestion
      <LinkerArg Include="/WHOLEARCHIVE:$(BootstrapperLibraryName).lib" />
```
to follow casing convention for link.exe options.